internal/handlers/user/handlers_test.go

```diff
@@ -511,7 +511,7 @@ func TestEnableTFA(t *testing.T) {
 		hw.um.AssertExpectations(t)
 	})
 
-	t.Run("enble tfa succeeded", func(t *testing.T) {
+	t.Run("enable tfa succeeded", func(t *testing.T) {
```

Comment:
Not sure what TFA is, normally I'd expect to see 2FA.

Reply:
We started using it in a few places as a replacement for 2fa in variables that had to start with a letter. And we we probably got used to it (and abused it ;). I'd swear I've seen this one around though (does not mean it's correct of course).

I don't mind if you want to update this particular entry in the test, but let's leave the `tfa` occurrences in variables and functions please.